### PR TITLE
fix: issue when no-one active

### PR DESCRIPTION
**Pull request by rachadiannovansyah**
### Overview
- fix(pop-up-banners): issue when no one active

### Related to the ticket
https://app.clickup.com/t/860pjr5yx

cc: @jabardigitalservice/jds-backend 

Evidence
project: Portal Jabar
title: Fix issue when no one pop-up banner active on update list CMS Portal Jabar
participants: @rachadiannovansyah @rindibudiaramdhan @ayocodingit @rhmnmbr @muhamadabduh

### CODE REVIEW

**Review by rhmnmbr — APPROVED** [review]

File: core-service/src/modules/pop-up-banner/repository/mysql/mysql_pop_up_banner.go [hunk 1]
```diff
@@ -119,14 +119,15 @@ func (m *mysqlPopUpBannerRepository) GetByID(ctx context.Context, id int64) (res
 	return
 }
 
-func (m *mysqlPopUpBannerRepository) CheckStatus(ctx context.Context, status string) (id int64, err error) {
+func (m *mysqlPopUpBannerRepository) CheckStatus(ctx context.Context, status string) (id int64, isActive bool) {
```
Comment (ayocodingit) [inline core-service/src/modules/pop-up-banner/repository/mysql/mysql_pop_up_banner.go:122]: how to handle an error fatal from SQL, if don't return an error message 😞

**Review by muhamadabduh — APPROVED** [review]

File: core-service/src/modules/pop-up-banner/repository/mysql/mysql_pop_up_banner.go (hunk 1 above)
Comment (rachadiannovansyah) [reply core-service/src/modules/pop-up-banner/repository/mysql/mysql_pop_up_banner.go]: fixed here [refactor: repository deactive status pop up banner](https://github.com/jabardigitalservice/portal-jabar-services/pull/366/commits/eb74c9fb4dfd2d3bba13eef843e038c9f9d1fa00) mate @ayocodingit

**Review by rindibudiaramdhan — APPROVED** [review]
Nice! Deactive all first, then active the selected banner.

**Review by ayocodingit — APPROVED** [review]

File: core-service/src/modules/pop-up-banner/repository/mysql/mysql_pop_up_banner.go (hunk 1 above)
Comment (ayocodingit) [reply core-service/src/modules/pop-up-banner/repository/mysql/mysql_pop_up_banner.go]: thanks, @rachadiannovansyah

**Pull request merged**